src/utils/download.ts

```diff
@@ -0,0 +1,25 @@
+import * as fs from 'fs';
+import path from 'path';
+import axios, { AxiosResponse } from 'axios';
+
+/**
+ * Downloads file from remote HTTP[S] host and puts its contents to the
+ * specified location.
+ * @param url URL of the file to download.
+ * @param destinationPath Path to the destination file.
+ */
+export async function download(url: string, filePath: string): Promise<string> {
+	const fileName = url.split('/').pop();
+	if (!fileName) {
+		throw new Error('Invalid remote path. No file name found.');
+	}
+	try {
+		const response: AxiosResponse = await axios.get(url, { responseType: 'arraybuffer' });
+		const pathToFile = path.join(filePath, fileName);
+
+		fs.writeFileSync(pathToFile, response.data);
+		return pathToFile;
+	} catch (error) {
+		throw new Error(`Failed to download file from remote path ${url}: ${error.message}`);
```

Comment:
non-blocking nit: We could gain test coverage over this error by exposing `axiosInstance` as an optional parameter and stub it during the tests with `stub(axiosInstance, 'get').resolves(throw Error('ERROR'))` (or similar, that may not be syntactically correct)